src/Shared/BenchmarkRunner/Program.cs

```diff
@@ -62,12 +62,12 @@ private static int Main(string[] args)
 
         private static IConfig GetConfig()
         {
-#if NET5_0 || NET6_0
+#if NET5_0 || NETCOREAPP5_0
             return ManualConfig.CreateEmpty()
                 .AddJob(Job.Default
                     .WithToolchain(CsProjCoreToolchain.From(new NetCoreAppSettings
                     (
-                        // not using "net6.0", a workaround for https://github.com/dotnet/BenchmarkDotNet/pull/1479
+                        // not using "net5.0", a workaround for https://github.com/dotnet/BenchmarkDotNet/pull/1479
```

Comment:
huh❔ this _does_ use `net5.0`

Reply:
I think you're misreading this or misunderstanding. BenchmarkDotNet has/had an issue where setting `targetFrameworkMoniker` to `net5.0` didn't work. So this comment is explaining that.

This was accidentally modified when Will did the NET6_0 changes.

Reply:
Ah, I see. It's using `targetFrameworkMoniker: "netcoreapp5.0"` versus `targetFrameworkMoniker: "net5.0"`. Not about `net5.0` instead of `net6.0`.

Reply:
Guess since we have a new BDN version we don't need the workaround anymore :D